### ISSUE TITLE
validate_required raise expection when value is not a string

### ISSUE THREAD

**Issue by pierreabreup** [sticksnleaves/justify#8]
This data
```
%{
  "identifier" => %{"propertyId" => "", "typeOf" => "", "value" => ""},
  "typeOf" => ""
}
```
raise a exception because `identifier` is not a string

```
** (FunctionClauseError) no function clause matching in String.Break.trim_leading/1
        (elixir) lib/elixir/unicode/properties.ex:288: String.Break.trim_leading(%{"propertyId" => "", "typeOf" => "", "value" => ""})
        (elixir) lib/string.ex:1098: String.trim/1
        (justify) lib/justify/validators/required.ex:17: anonymous fn/5 in Justify.Validators.Required.call/3
        (elixir) lib/enum.ex:1925: Enum."-reduce/3-lists^foldl/2-0-"/3
```

**Comment by pierreabreup** [comment]
I will try to fix it next weekend

**Comment by anthonator** [comment]
Fixed in 19aed1f5c5dabe57ec0b53601722e671d8fbc45d
